lib/readyset.rb

```diff
@@ -88,7 +89,7 @@ def self.drop_cache!(name_or_id: nil, sql: nil)
   # @param sql_array [Array<Object>] the SQL array to be executed against ReadySet.
   # @return [PG::Result] the result of executing the SQL query.
   def self.raw_query(*sql_array) # :nodoc:
-    ActiveRecord::Base.connected_to(role: reading_role, shard: shard, prevent_writes: false) do
+    ActiveRecord::Base.connected_to(role: writing_role, shard: shard, prevent_writes: false) do
```

Comment:
An unrelated bug that needed fixing. Our internal `raw_query` method needs to allow writes so we can invoke `CREATE CACHE` and `DROP CACHE` on ReadySet